.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 🚀 Feature
+about: Ask for a well defined and described Feature for CMaNGOS.
+---
+
+## 🚀 Feature
+<!-- Describe the Feature. -->
+
+### Visualisation
+<!-- Attach Screenshots and Drawings. Use https://draw.io -->
+<!-- Specify more details of the Feature with each Picture. -->
+
+### Schedule
```

Comment:
Instead of Schedule can we put something like 'why you want this feature added and what benefits it would provide'?

Reply:
https://github.com/ulfgebhardt/issues/issues/7